2dfft_utils/analysis/spiral_overlay.py

```diff
@@ -0,0 +1,276 @@
+'''
+    spiral_overlay.py
+    
+    DESCRIPTION:        Overlay spirals onto a FITS image.
+    
+    USAGE:              python spiral_overlay.py <filename>
+    
+    INPUTS:             Filename.
+                        FITS filename can be passed as a command line argument,
+                        but the program will prompt for it if it isn't.
+    
+    OUTPUT:             Interactive plot in python window.
+    
+    NOTES:              Uses pylab (including matplotlib and numpy) and astropy.
+                        All necessary libraries included in Ureka package.
+    
+                        Eqns used:
+    
+                            r = ae^(b*theta)
+    
+                            x = r*cos(theta)
+                            y = r*sin(theta)
+    
+                            tan(phi) = b
+                            phi = pitch angle
+    
+                        Edit theta_max, title, origin of the plot (imshow)
```

Comment:
This is probably not something users need to edit in the source code any longer since they're working interactively instead of from the command line.